docs/contributing/triaging-github-issues.md

```diff
@@ -3,7 +3,103 @@ title: Triaging GitHub Issues
+## What is the aim of this document?
 
-Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+As a Gatsby core team, we've found patterns that help us effectively triage incoming Github issues, providing answers to the community's questions, identify bugs as well as opportunities to contribute to Gatsby core. Triaging issues is a great way to contribute to the Gatsby community and share your knowledge, without necessarily requiring a lot of deep context about the way the Gatsby codebase works.
+
+We want to share these patterns with the broader community, so that if you're interesting in helping us triage, you're able to do so more effectively!
+
+This document should answer any questions you might have along the way. We’ll answer common questions, list guidelines and illustrate a decision tree.
+
+## What is first touch maintenance?
+
+For Gatsby the first line of communication between a user and the team is the issue tracker on GitHub. Typically, every day 20-30 issues are opened -- that's one every hour!
+
+These issues could be:
+
+- a question that can be answered immediately
+- a bug report
+- a request for a feature
+- or a discussion on a complicated use case
+
+As a first touch maintainer, you might sift through, triage, communicate and manage this first line of communication.
+
+They typically:
+
+- answer questions that can be answered immediately
+  -test and reproduce possible bug reports and label them appropriately
+- communicate feature requests to the rest of the team and ensure a valid response
+- enable discussions on complicated use cases, whether themselves or via the rest of team
+
+## Why do we do first touch maintenance?
+
+We do first touch maintenance so that:
+
+- Questions are answered swiftly and correctly, therefore making users happy
+- Bug reports are reproducible and the most relevant data is collected before someone jumps in with a fix
+- Unrelated issues are resolved swiftly so we don't spend too much time on them
+
+## What is unique to Gatsby?
+
+Gatsby is unique among most open source projects because:
+
+- Gatsby integrates with many third party tools (WordPress, Drupal, Contentful etc) via source plugins and hence the typical scope of issues is widened significantly
+- Gatsby aims to be really beginner friendly (we want to be the new way someone gets started with web development) and this means that we need to accommodate a wide range of skill levels
+- At Gatsby, we've defined a couple of metrics that we measure to ensure we're responsive and helpful to our open source community
+
+## How do we do first touch maintenance?
+
+### General Guidelines
+
+- **Be empathic.** The author of an issue might be asking something that’s obvious to you but that doesn’t mean it’s obvious to them - it's important to consider the issue from the author’s viewpoint. People often remember how you make them feel, not what you told them.
```

Comment:
```suggestion
- **Be empathetic.** The author of an issue might be asking something that’s obvious to you but that doesn’t mean it’s obvious to them - it's important to consider the issue from the author’s viewpoint. People often remember how you make them feel, not what you told them.
```

Reply:
I looked that up too as I'm more used to "empathetic", but "empathic" works here!

Reply:
I prefer empathetic too (empathic was a recommendation from @m-allanson)

Let's move to it since it seems to be more widely used!